File: src/main/java/me/andre111/d20server/message/util/ResponseFail.java
package me.andre111.d20server.message.util;

import me.andre111.d20server.message.Message;

public class ResponseFail extends Message {
	public final String to;
	public final String description;
	
	public ResponseFail(Message message, String description) {
		this.to = message.getClass().getSimpleName();
		this.description = description;
	}
}

File: core/common/message/util/response-fail.js
// @ts-check
import { Message } from '../message.js';
import { registerType } from '../../util/datautil.js';

export class ResponseFail extends Message {
    to;
    description;

    constructor(to, description) {
        super();
        this.to = to;
        this.description = description;
    }

    getTo() {
        return this.to;
    }

    getDescription() {
        return this.description;
    }
}
registerType(ResponseFail);
